release/update-rsync.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Update rsync server directories with latest bundled source code
+# This will update rsync to the latest snapshot OR release tarball
+# available in the www service
+#
+
+test -e ~/.server.config && . ~/.server.config || exit 1
```

Comment:
AFAICT, this can and should be significantly improved to:

* provide a useful error message instead of failing in complete silence when the configuration file is missing (see below for sample output)
* simplify
* avoid misleading a reader with what looks like "load an optional configuration file if it exists" logic but is actually "load a required configuration file" logic
* remove code duplication


```suggestion
. ~/.server.config || exit 1
```

Or, if we also want to preserve the failure error code:

```suggestion
. ~/.server.config || exit $?
```

Please propagate this change to other scripts that load configuration this way.

----

Before the suggested change, there is no output on failures due to missing configuration:

```sh
$ test -e ~/.server.config && . ~/.server.config || echo "exit $?"
exit 1
```

After the suggested change, there is useful output:

```sh
$ . ~/.server.config || echo "exit $?"
bash: /homer/tester/.server.config: No such file or directory
exit 1
```

Reply:
Out of scope request. These changes are the scope of PR #7.

Reply:
see https://github.com/squid-cache/ci/pull/7/commits/27dd745519680ec102b2297f3cead55623e86cb4